packages/frontend/src/stores/messagelist.ts

```diff
@@ -981,6 +981,10 @@ class MessageListStore extends Store<MessageListState> {
             `even belong to chat ${chatId}? Or did the message get deleted?\n` +
             `Anyways, falling back to jumping to the last message.`
         )
+        window.__userFeedback({
+          type: 'error',
+          text: `${window.static_translate('error')}: message not found`,
```

Comment:
For the next time: we have the `error_x` string, which should be more appropriate.